sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayoutController.java

```diff
@@ -181,9 +181,15 @@ public void setFullscreenButtonShowing(boolean showing) {
 
   @ReactMethod
   public void onClosedCaptionUpdateRequested(ReadableMap parameters) {
-    final String languageName = parameters.hasKey("language") ? parameters.getString("language") : null;
-    double curTime = _player.getPlayheadTime() / 1000d;
-    WritableMap params = BridgeMessageBuilder.buildClosedCaptionUpdateParams(_player, languageName, curTime);
+    WritableMap params;
+    // ignore the request if cc is not available
+    if (_player == null || _player.getCurrentItem() == null || !_player.getCurrentItem().hasClosedCaptions()) {
+       params = Arguments.createMap();
```

Comment:
Any chance we can clear the state in a good way in the skin, without clearing the CC selection? It sounds like we _should_ be clearing the CC text when the skin gets the new video.